Tools/autotest/param_metadata/rstemit.py

```diff
@@ -24,14 +24,17 @@ def toolname(self):
 
     def __init__(self, *args, **kwargs):
         Emit.__init__(self, *args, **kwargs)
-        output_fname = 'Parameters.rst'
+        if self.latexpdf:
+            output_fname = 'ParametersLatex.rst'
+        else:
+            output_fname = 'Parameters.rst'
         self.f = open(output_fname, mode='w')
```

Comment:
```suggestion
        self.f = open(self.output_fname(), mode='w')
```

and outside the init function:
```
    def output_fname(self):
        return 'Parameters.rst'
```

... and that's overridden by the new class.